testing/jattach/APKBUILD

```diff
@@ -0,0 +1,24 @@
+# Contributor: Andrey Pustovetov <andrey.pustovetov@gmail.com>
+# Maintainer: Andrey Pustovetov <andrey.pustovetov@gmail.com>
+pkgname=jattach
+pkgver=1.0
+pkgrel=0
+pkgdesc="JVM dynamic attach utility"
+url="https://github.com/apangin/jattach"
+arch="all"
+license="Apache-2.0"
+makedepends="gcc"
```

Comment:
This shouldn't be needed since abuild pulls in `build-base` automatically which depends on gcc.

Reply:
Thank you. I removed `makedepends` variable, CI build passed.